src/js/edu-benefits/5490/helpers.jsx

```diff
@@ -8,3 +8,8 @@ export function transform(formConfig, form) {
     }
   });
 }
+
+export const benefitsLabels = {
+  chapter35: 'DEA (Chapter 35)',
+  chapter33: 'Fry scholarship (Chapter 33)'
```

Comment:
I would guess so, but yeah, we don't have the content for them as far as I know.